crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -396,6 +401,7 @@ pub struct AuthorityEpochTables {
     /// the sequence number of checkpoint does not match height here.
     #[default_options_override_fn = "pending_checkpoints_table_default_config"]
     pending_checkpoints: DBMap<CheckpointCommitHeight, PendingCheckpoint>,
+    pending_checkpoints_v2: DBMap<CheckpointCommitHeight, PendingCheckpointV2>,
```

Comment:
done re: the override_fn

which thing are you recommending I rename with #16366?

Reply:
you could do:  

```
#[rename = "pending_checkpoints"]
pending_checkpoints_deprecated: ...
#[rename = "pending_checkpoints_v2"]
pending_checkpoints: ...
```

this way the name of the table in rust wouldn't change, it would only change the cf names